source/user-manual/registering/index.rst

```diff
@@ -1,63 +1,127 @@
+
+
+
+ .. group-tab:: Windows host
+
+   Open a Powershell or CMD session in the Wazuh agent's host as an ``Administrator``.
+
+    .. include:: ../../_templates/registrations/windows/installation_directory.rst
+
+   1. To register the Wazuh agent, run the ``agent-auth.exe`` utility, using the Wazuh manager's IP address:
+
+    .. code-block:: console
+
+     # C:\Program Files (x86)\ossec-agent\agent-auth.exe -m <manager_IP>
+
+    .. include:: ../../_templates/registrations/common/set_agent_name.rst
+
+   2. To enable the communication with the Wazuh manager, edit the Wazuh agent's ``C:\Program Files (x86)\ossec-agent\ossec.conf`` configuration file:
+
+    .. include:: ../../_templates/registrations/common/client_server_section.rst
+
+   3. Restart the Wazuh agent:
+
+    .. include:: ../../_templates/registrations/windows/restart_agent.rst
+
+   The Wazuh agent registration can be adjusted by using different :ref:`agent-auth` options.
+
+
+
+ .. group-tab:: MacOS X host
+
+  Open a terminal in the MacOS X Wazuh agent's host as a ``root`` user.
+
+  1. To register the Wazuh agent, run the ``agent-auth`` utility, using the Wazuh manager’s IP address:
+
+   .. code-block:: console
+
+    # /Library/Ossec/bin/agent-auth -m <manager_IP>
+
+   .. include:: ../../_templates/registrations/common/set_agent_name.rst
+
+  2. To enable the communication with the Wazuh manager, edit the Wazuh agent's ``/Library/Ossec/etc/ossec.conf`` configuration file:
+
+   .. include:: ../../_templates/registrations/common/client_server_section.rst
+
+  3. Restart the Wazuh agent:
+
+   .. include:: ../../_templates/registrations/macosx/restart_agent.rst
+
+  The Wazuh agent registration can be adjusted by using different :ref:`agent-auth` options.
+
+The above method is the simplest way of registering the Wazuh agents. There are also available other registration methods:
+
++-----------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Registration method                                                                                 | Description                                                                                                                                                  |
++=====================================================================================================+==============================================================================================================================================================+
+| :ref:`Using command line (CLI) <command-line-registration>`                                         | Manual registeration using ``manage_agents`` utility. Requires extracting the registration key from the Wazuh manager and inserting it in the Wazuh agent.   |
```

Comment:
Inserting it manually in the Wazuh agent.

Reply:
Done